src/Tx_main.cpp

```diff
@@ -75,7 +75,21 @@ void RebootIntoWifi()
 
 void ProcessMSPPacketFromPeer(mspPacket_t *packet)
 {
+  if (packet->function == MSP_SET_NAME) {
```

Comment:
The problem with this is that 11 (MSP_SET_NAME) is also 0xB (MSP_ELRS_REQU_VTX_PKT) So this change is going to break the VRX backpack requesting the VTX channel from the transmitter.

Reply:
I see. I just picked 11 since it is the command is betaflight used for the set name function.

Do you have a suggestion for an alternative value?

Reply:
Yeah, I would make it 0x12 and move it just above MSP_ELRS_SET_OSD.
You will then have to either change the PR for the main ELRS firmware to rewrite the MSP packet or rewrite the packet here before sending it back to the main ELRS firmware. I prefer the second option as it has less impact on the main firmware.

Ultimately I think we should really should change the code for the ELRS specific MSP messages but that would mean all 3 firmwares would need to be updated!

Reply:
@pkendall64 should be resolved now :)